### PR TITLE
#8: Update to Yapsy 1.11.223 and YAML

### PR DESCRIPTION
**Pull request by OOPMan**
PR for #8. Paging Chief Data Violator @fluffels to the PR!

### CODE REVIEW

File: src/jormungand/__init__.py [hunk 1]
```diff
@@ -13,111 +12,147 @@
...
+                if isinstance(path, (str, unicode))
+            ]
+            self.getPluginLocator().plugins_places.extend(plugin_roots)
+        self.extendPluginPlaces([os.path.join(os.path.abspath(os.path.dirname(__file__)), 'plugins')])
 
-    def extendPluginPlaces(self, directories_list):
-        """ Extend the current list of plugin locations with new entries """
-        self.plugins_places.extend(directories_list)
+    def extendPluginPlaces(self, paths):
+        self.getPluginLocator().plugins_places.extend(paths)
```
Comment (fluffels) [inline src/jormungand/__init__.py]: @OOPMan what is the point of this function?

File: src/jormungand/__init__.py [hunk 2]
```diff
@@ -13,111 +12,147 @@
...
+                                    elif isinstance(plugin_constructer_args, dict):
+                                        plugin_info.plugin_object = element(**plugin_constructer_args)
+                                    else:
+                                        plugin_info.plugin_object = element()
+                                        plugin_info_reference = plugin_info
+                                except Exception:
+                                    exc_info = sys.exc_info()
+                                    log.error("Unable to create plugin object: {}".format(candidate_filepath), exc_info=exc_info)
+                                    plugin_info.error = exc_info
+                                    break  # If it didn't work once it wont again
```
Comment (fluffels) [inline src/jormungand/__init__.py]: @OOPMan isn't it possible that the next one will work? also, isn't it a good idea to just bail here in general?

Comment (OOPMan) [inline src/jormungand/__init__.py]: @fluffels This is the behaviour of Yapsy so....no clue? :-)

Comment (fluffels) [inline src/jormungand/__init__.py]: @OOPMan yuck

**Pull request merged**